handout/aot/aot_limitations.md

```diff
@@ -0,0 +1,4 @@
+# AoT limitations
+However, AoT is not perfect. The main limitation is that AoT, due to the way it compiles the raw code, cannot be used with common code patterns, for example, default exports from modules, template literals for templates, and functions in providers, routes, or declarations. Currently, we do not have a complete list of "AoT Do's and Don'ts" and the Angular team has not released anything regarding this issue. Rangle made its own list [here](https://github.com/rangle/angular-2-aot-sandbox) and also provides a sandbox for testing features with AoT.
```

Comment:
If we are going to be pointing people to the AoT test repo, I think we need to create snippets of "Don't" and "Do" there to be easier to understand the limitations (See [Issue #5](https://github.com/rangle/angular-2-aot-sandbox/issues/5))